.github/workflows/release.yml

```diff
@@ -10,6 +10,30 @@ on:
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
+  cli-install-cross-platform-release-test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [18, 20, 21, 22, 23]
```

Comment:
why no 19 - also is 24 out yet i can't recall

Reply:
I'll be honest as i never used v19 i completely forgot about it 🤣

Reply:
btw no v24 yet https://nodejs.org/en/about/previous-releases#looking-for-the-latest-release-of-a-version-branch